qiskit/circuit/qpy_serialization.py

```diff
@@ -186,7 +186,10 @@ class if it's defined in Qiskit. Otherwise it falls back to the custom
 utf8 data for the label if one was set on the instruction. Following the label
 bytes if ``has_conditional`` is ``True`` then there are
 ``conditonal_reg_name_size`` bytes of utf8 data for the name of the condtional
-register name.
+register name. In case of single classical bit conditions the register name
+utf8 data will be prefixed with a null character "\x00" and then a utf8 string
```

Comment:
```suggestion
utf8 data will be prefixed with a null character "\\x00" and then a utf8 string
```

or make the docstring raw, unless we actually want to embed the null byte.  I think it'll mess up the `help()` docstring (`less` doesn't play so nicely with them), and Sphinx won't show it.

Reply:
Oops, yeah good catch